### ISSUE TITLE
Author 1871628 crashes record editor

### ISSUE THREAD

**Issue by michamos** [inspirehep/inspirehep#2025]
See https://sentry.inspirehep.net/inspire-prod/record-editor/issues/36341/?query=is%3Aunresolved and https://inspirehep.zulipchat.com/#narrow/stream/122202-devel/topic/author.20not.20loading.20in.20record.20editor/near/245004982.

**Comment by michamos** [comment]
Similar story for https://inspirehep.net/authors/1701369. One possible reason is that the editor expects `positions.record` to be present, but it's often not the case for recent edits due to #2026.

**Comment by hoc3426** [comment]
Same with
https://inspirehep.net/authors/1877090

**Comment by hoc3426** [comment]
Margaret reports:
It's already been discovered that you cannot edit, hit save, and then stay on the same page and start editing again.
However, even when I edit, hit save - if I don't get that save message in the lower right-hand corner, and just
exit the page, when I go back in, I get the grey screen of death - for any author I am trying to look up. When I get
the "grey" screen of death in the editor, it lasts for @ 20 minutes. 

However, this is what I'm trying to discern - what is causing the editor not to save very basic edits - adding a new
institution, id, email, advisor.

One cause of the grey screen of death is from hitting return within the editor while actively editing. As soon as you do that,
you cannot add anymore fields.

EXAMPLE: Hit big plus to add a new institution, paste in the institution name copied from the inst database. The institution
name will appear blue in the editor. I then hit enter. 

Next I try to add another field e.g. email, nothing happens when you hit the big plus on the upper left-hand side. Since I
can't add any additional fields I hit save (so as to not lose inst entry). The green success window in the lower right-hand side
does not appear. 

With no success window, I just exit the page. Upon refreshing the author's profile, the new institution I added will appear, but
in black, not blue like I saw in the editor meaning no link to the inst database.

As soon as I can get into another editor, I will try only tabbing throughout the editor and see if that works. However, tabbing,
hitting enter or clicking outside the editor should not disable the editor from working.
